src/js/widget-layout-editor.js

```diff
@@ -164,7 +164,7 @@ class WidgetLayoutEditor extends React.PureComponent {
         });
         this.setState({widgets: widgets});
         if (changed) {
-            this.updateAllServerWidgets();
+            this.updateServerWidgets();
```

Comment:
Keep as updateAllServerWidgets.  The function you left is updateAllServerWidgets.